src/internal_events/lua.rs

```diff
@@ -20,7 +20,7 @@ pub struct LuaGcTriggered {
 
 impl InternalEvent for LuaGcTriggered {
     fn emit_metrics(&self) {
-        gauge!("memory_used", self.used_memory as i64,
```

Comment:
Why are all these integers turning into floats?

Reply:
A change in `metrics` crate. I don't think it matters for our use cases here.